mmdeploy/codebase/mmdet/models/utils/gaussian_target.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmdet.models.utils.gaussian_target.get_topk_from_heatmap')
+def get_topk_from_heatmap__default(scores, k=20):
```

Comment:
tensorrt-fp16 is good, but tensorrt-fp16+sdk failed with performance. box mAP 25.8(pytorch) vs 0.5 (SDK)

```
python tools/regression_test.py --codebase mmdet --models CenterNet --performance --work-dir ./workdir --backends tensorrt
```